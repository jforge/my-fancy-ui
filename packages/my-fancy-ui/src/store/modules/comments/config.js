// @flow

export const namepsace = '@immowelt/my-fancy-ui';
export const moduleId = 'comments';
