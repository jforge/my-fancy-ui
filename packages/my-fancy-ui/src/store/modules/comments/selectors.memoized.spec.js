import * as selectors from './selectors.memoized.js';
import {moduleId} from './config.js';

describe('getCommentIds()', () => {
  it('should be a function.', () => {
    expect(typeof selectors.getCommentIds).toBe('function');
  });

  it(`should return the "${moduleId}.commentsById" value.`, () => {
    const state = {
      [moduleId]: {
        commentsById: {
          foo: {},
          bar: {}
        }
      }
    };
    const result = selectors.getCommentIds(state);

    expect(result).toMatchSnapshot();
  });
});
