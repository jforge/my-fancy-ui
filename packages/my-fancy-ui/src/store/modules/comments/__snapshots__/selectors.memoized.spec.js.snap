// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getCommentIds() should return the "comments.commentsById" value. 1`] = `
Array [
  "foo",
  "bar",
]
`;
